tests/integration/devfile/cmd_devfile_storage_test.go

```diff
@@ -158,17 +159,17 @@ var _ = Describe("odo devfile storage command tests", func() {
 			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-with-volume-components.yaml"), filepath.Join(commonVar.Context, "devfile.yaml"))
 
 			stdOut := helper.CmdShouldPass("odo", "storage", "list", "--context", commonVar.Context)
-			helper.MatchAllInOutput(stdOut, []string{"firstvol", "secondvol", "Not Pushed", "CONTAINER", "runtime", "runtime2"})
+			helper.MatchAllInOutput(stdOut, []string{"firstvol", "secondvol", "/secondvol", "Not Pushed", "CONTAINER", "runtime", "runtime2"})
```

Comment:
since we are now checking for the path `/secondvol`, it might be a good idea to also check that the other paths (`/data`, `/data2`) are being displayed too

Reply:
Updated, I've also added a new commit for storage commit changes